site/src/scaffold/styles/layout.scss

```diff
@@ -102,9 +102,11 @@ $size-bar-width: 300px - strip-units($gap-size) * $document-font-size;
 
 #article {
 	grid-area: article;
+	max-width: 100%;
```

Comment:
Notice this is required to ensure the main content area does not overflow.

Reply:
seems to work without it- specific resolution needed?